### PR TITLE
Feat: added method get_distribution for class ActorCriticPolicy

### PR DESCRIPTION
**Pull request by cyprienc**
## Description
New method get_distribution for class ActorCriticPolicy returning current action distribution given observations

## Motivation and Context
This is related to https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/pull/40#discussion_r706595367
TRPO will require the Distribution of the current policy.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:

- [x] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)
- [x] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

File: docs/misc/changelog.rst [hunk 1]
```diff
@@ -4,6 +4,31 @@ Changelog
...
+Release 1.2.1a0 (WIP)
+---------------------------
+
+
+Breaking Changes:
+^^^^^^^^^^^^^^^^^
+
+New Features:
+^^^^^^^^^^^^^
+- Added method `get_distribution` for `ActorCriticPolicy` (@cyprienc)
```
Comment (araffin) [inline docs/misc/changelog.rst:16]:
```suggestion
- Added method `get_distribution` for ``ActorCriticPolicy`` (@cyprienc)
```

**Review by araffin — COMMENTED** [review]
Only a test is missing, otherwise LGTM ;)

**Comment by araffin** [comment]
I added the test and fixed the changelog in https://github.com/DLR-RM/stable-baselines3/pull/566
but I couldn't push the changes to your PR because you were using master branch... so I opened another PR.

**Comment by cyprienc** [comment]
Ok - thanks a lot - I am closing this one then

**Pull request closed without merging**